test/response/api/v1/attendances/get.json

```diff
@@ -1,337 +1,504 @@
 {
   "items": [
     {
+      "id": 41761,
+      "team": {
+        "id": 3,
+        "href": "http://amam.sample.com/api/v1/teams/3"
+      },
+      "member": {
+        "id": "100606116",
+        "href": "http://amam.sample.com/api/v1/members/100606116"
+      },
+      "attendanceType": {
+        "id": 5,
+        "href": "http://amam.sample.com/api/v1/attendance_types/5"
+      },
+      "attendanceReason": {
+        "id": 6,
+        "href": "http://amam.sample.com/api/v1/attendance_reasons/6"
+      },
+      "remarks": "中国",
+      "createdAt": "2016-08-25T19:01:19.000+09:00",
+      "updatedAt": null,
+      "date": "2016-12-26",
```

Comment:
Exactly, it should not depend time zone but we need more consideration and discussion.a
Currently "date" is one of composite unique key of Attendance model and it has index also. So, if changed start/end model, system should be changed significantly.

Not changed key and just changes format with 0 pad like "YYYY-MM-DD T00:00:00.000+09:00" is no problem at this moment I think.